tests/integration/test_charm.py

```diff
@@ -171,3 +174,52 @@ async def test_invalid_jvm_config(ops_test: OpsTest) -> None:
                 "Invalid jvm-config, "
                 "Java heap memory specification exceeds application memory constraint"
             )
+    await asyncio.gather(
+        *(
+            [
```

Comment:
Could this be a generator? Can also use `itertools.chain`

Reply:
Here are just 4+1 elements, `itertools.chain` doesn't provide much more clarity here.